### DISCUSSION TITLE
Lerner discussed on the Joe Rogan podcast

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#129] [General]
From 4 weeks ago
https://youtube.com/shorts/mRdiAsNcD_Q

Rogan has a hunch the Big Bang is bunk.
@12:00 https://youtu.be/tb1Ubw1Iu5w

**Comment by HanDeBruijn** [comment]
I would suggest to skip the first video and play the second one from the beginning to the end.
It is enlighting to see how the whole (academic) world has got involved with the BB ideology, Stephen Hawking with his Black Hole ideas, supported by the mathemacist (aka physicist) [Roger Penrose](https://en.wikipedia.org/wiki/Roger_Penrose), resulting in stuff like the [Penrose–Hawking singularity theorems](https://en.wikipedia.org/wiki/Penrose%E2%80%93Hawking_singularity_theorems). As if singularities do really exist in physics. Penrose even got (half) [The Nobel Prize in Physics 2020](https://www.nobelprize.org/prizes/physics/2020/summary/) for this work.
I am not at all impressed by the person. [The Emperor's New Mind](https://en.wikipedia.org/wiki/The_Emperor%27s_New_Mind) must be somewhere in my bookshelves (Dutch translation). You can have it for free, if I can find it back.

**Reply by HanDeBruijn** [reply]
![Penrose](https://github.com/a-cosmology-group/acg/assets/135433222/1c748d14-53b4-41e5-beea-cd81d086e533)

**Comment by MJoseph315** [comment]
Isn't Joe Rogan a climate change denier? A climate change denier who once hosted alt-right nutjob Jordan Peterson? Don't get me wrong, I agree with many of Eric Lerner's critiques of the big bang theory, I just don't think we should be courting approval from people like Rogan. If anything, I think getting approval from people like him will only encourage people to mistrust us and our ideas.

**Reply by bgreenman** [reply]
i agree, the NY Times article and attending-hosting forums in scientific conferences is a better way to obtain legitamscy

**Comment by sahil5d** [comment]
Whoa. Bill Maher has doubts about the Big Bang Theory too.
https://youtu.be/EopP1iD6N3w?si=vj9d1jsT83zoxo1N&t=364

**Reply by bgreenman** [reply]
Good, but the best place to discuss these topics is in intellectual forums

**Reply by RedshiftDrift** [reply]
Everybody has doubts about the Big Bang model, but not cosmologists.  See how quickly the LA Times was dismissing the BB in 1941.
However, when scientists wash their dirty laundry in public they lose credibility (c.f. COVID-19, global warming, etc.) so let's not do that with cosmology.
![LATimes_SavantRefutesTheoryExplodingUniverse_1280](https://github.com/a-cosmology-group/acg/assets/16748402/bf646b5e-4557-4c93-8f0c-addbdfac0d60)

**Reply by budrap00** [reply]
Science loses its credibility when scientists present science as a body of irrefutable knowledge with themselves as omniscient purveyors of that knowledge. Admitting and correcting mistakes is an integral part of the scientific process. Science is no more infallible than the Pope. 

The pretense that it is infallible is what damages scientific credibility. If you are suggesting that the "washing of dirty laundry" (public discussion of scientifically contentious issues) harms science I would argue just the opposite. Science cannot and should not be presented to the public as anything but what it is - a well established method for investigating the nature of physical reality which inherently entails debate, discussion and disagreement. It cannot be presented as something it is not - a repository of fixed, unassailable truths.

**Reply by bgreenman** [reply]
Agree with your statement, science is a process not a dogma

**Reply by RedshiftDrift** [reply]
By "washing of dirty laundry in public" I mean the publicly discussed ***technical*** aspects of science.  This harms science because the public does not understand what is being debated.  Bill Maher has no clue of what's wrong with the Big Bang model, he just finds the idea of *"a universe the size of a marble"* difficult to accept.  This is wrong because Maher says "science is difficult to accept", presenting it as the "truth" rather than saying science is a well established method for investigating physical reality (not the "nature of realty" by the way, philosophy does that).

Science should always be presented to the public in a simplified form understandable by the non-scientist.  Broadcasting things like [*"I put my faith in deGrasse Tyson's "genius"*](https://youtu.be/EopP1iD6N3w?t=364), ["the universe doesn't have to make sense to you"](https://youtu.be/EopP1iD6N3w?t=385) or ["I have a view that is factual"](https://youtu.be/EopP1iD6N3w?t=414) is absolutely wrong because non-scientists don't see the world that way.  It harms science when people like Maher let scientists publicly say "that's how I see the world because I am a scientist and you, the public, don't do it the right way".

I blame science journalists and ignorant TV hosts for reporting on the sensational without explaining why it is sensational.  The 1941 article above did explain why Hubble believed his observations falsified the Big Bang, and Carl Sagan used to do a great job.  But with time the Big Bang model has grown more complex and the public has lost track of what the theory really says.  When I am with non-scientists and someone asks me what I do, I ask them to describe the "Big Bang model" - their knowledge of the theory ranges from ***'very poor'*** to ***'null'***.  As a result, they can only parrot what they read online, and an explanation of what I do requires a long introduction to the history of the Big Bang model which I cannot deliver in 2 minutes.

**Reply by budrap00** [reply]
You can't have it both ways, hiding the technical aspects of science from the public and then complaining that the public and journalists etc. are ignorant of science. That doesn't mean that deep nuanced discussions have to be presented to people who are only casually interested but the general shape of the arguments can be regularly presented that leave open the possibility of deeper consideration. 

For instance its not that hard to explain to someone who is interested that a fundamental objection to the BB theory is that the Universe it describes is not observed in physical reality. If that's all the detail they can absorb then that's ok - you've presented in its simplest form the most fundamental problem with the standard model. 

You don't have to discuss redshift, CMB anisotropies, the failures of our gravitational models, plasma physics and so on, unless they are amenable to those deeper discussions. Sketching the nature of the BB and your objections to it is doable as long as you don't try to recount the arduous path by which you came to your current understanding. Tailoring your message to suit your audience isn't the same as dumbing it down to absolutist pronouncements. Science has a logical basis that most people can grasp if you present it to them in concrete physical terms.

**Reply by bgreenman** [reply]
Your ideas are correct

**Reply by RedshiftDrift** [reply]
> For instance its not that hard to explain to someone who is interested that a fundamental objection to the BB theory is that the Universe it describes is not observed in physical reality.

"Not hard"? Try to explain this to these people [who don't even have a conceptual grasp of astronomy](https://youtu.be/zOw8aiyMUAU).  Most people do not grab logic, they just learn what they're told.  Only knowledgeable people can figure out cosmology, not the public.  The public should be informed about the progress of our understanding of the universe (I never said to dumb it down), but not about the hurdles.

**Reply by ftessier** [reply]
Don't look up!! 😆

**Reply by budrap00** [reply]
Setting aside the fact that a random sample cherry-picked for comedic effect doesn't say much, let's allow your misanthropic view that the general population is wildly ignorant of astronomical matters. I cannot agree that the solution to that problem is to sequester scientific discussions from the public sphere. It seems to me that approach can only contribute to the general lack of scientific knowledge.

**Reply by ftessier** [reply]
I hold the belief that anything can be explained to anyone clearly (considering their level of knowledge, obviously). A shortcoming on that front is a failure on the part of the teacher, not the student. But of course, both mentor and mentee have to be genuinely interested and invested in the entreprise.

**Reply by RedshiftDrift** [reply]
Agreed! Before one can explain the reasons for discussing alternative models, the mentor 1) has to explain how science is done and 2) expain what the current model is.
The public is at step 1) but @budrap00 suggests to skip these steps and throw scientific discussions at an ignorant public... It is this failure to educate properly that I am highlighting, I do not advocate leaving the public in ignorance.  In fact I have given a few public talks explaining how to measure distance with the redshift of galaxies, and how this leads to inconsistencies that are still unresolved.  (My talks avoided the "argument from authority fallacy" used by Rogan, Maher and others.  When you don't explain the science, you are only left with arguments like "deGrasse Tyson says so and he is smart" which contribute to the lack of knowledge about the scientific method.)

**Reply by budrap00** [reply]
@RedshiftDrift  Taking both sides of an argument is one way of assuring you're always right, I guess.

`However, when scientists wash their dirty laundry in public they lose credibility (c.f. COVID-19, global warming, etc.) so let's not do that with cosmology.
`
`I do not advocate leaving the public in ignorance.
`

**Reply by HanDeBruijn** [reply]
@budrap00 . You get better quotes by using '&gt;' as in
```
>I do not advocate leaving the public in ignorance.
etcetera
```
giving
>I do not advocate leaving the public in ignorance. 
etcetera

Just a hint.

**Reply by budrap00** [reply]
Thanks.

**Comment by bgreenman** [comment]
The science press is a big part of the problem,since it covers science news differently than other topics(politics,culture,economics,etc...) by avoiding covering controversial debate on science topics. the general public gets the idea that the science field is a type of gee wiz exotic activity
